proxy/http2/Http2Frame.cc

```diff
@@ -76,11 +76,6 @@ Http2DataFrame::write_to(MIOBuffer *iobuffer) const
 int64_t
 Http2HeadersFrame::write_to(MIOBuffer *iobuffer) const
 {
-  // Validation
```

Comment:
I just got rid of this validation, because the `_hdr_block_len` is already checked before making `Http2HeadersFrame` and too noisy to read or pass the config from `ua_session` here.